src/App.js

```diff
@@ -6,14 +6,18 @@ import {
 } from 'react-router-dom';
 import Styled from 'styled-components';
 import About from './pages/About';
+import Footer from './components/Footer/Footer';
 import Home from './pages/Home';
 import Header from './components/Header/Header';
 import Search from './pages/Search';
 import HowItWorks from './pages/HowItWorks';
 
 const StyledWrapper = Styled.div`
+    position: relative;
+    width: 100%;
     max-width: 1440px;
     margin: 0 auto;
+    overflow-x: hidden;
```

Comment:
Why did you add this? If something is overflowing it might be a problem in the CSS

Reply:
Yes. it was before finding the problem.